File: api/BeHealth.Business/Resources/FoodManamgnet/FoodItems/FoodItemResource.cs
using BeHealth.Business.Resources.Settings;

namespace BeHealth.Business.Resources.FoodManamgnet.FoodItems
{
    public class FoodItemResource
    {
        public long ID { get; set; }

        public string Name { get; set; }

        public double Fat { get; set; }

        public double Carbohydrates { get; set; }

        public double Protein { get; set; }

        public double Calories { get; set; }

        public double BaseQuantity { get; set; }

        public int Group { get; set; }

        public UnitResource Unit { get; set; }

    }
}

File: app/src/app/features/food/resources/foodItem/food-item.resource.ts
import { TrackableResource } from 'src/app/core/resources/trackable.resource';
import { TypeResource } from 'src/app/features/settings/resources/types/type.resource';
import { UnitResource } from 'src/app/features/settings/resources/unit/unit.resource';
import { NutritionValueModel } from '../nutrition/nutrition-value.resource';

export class FoodItemResource extends TrackableResource {
  id: number;
  name: string;
  group: number;
  baseQuantity: number;
  unit: UnitResource;
  calories: number;
  fat: number;
  protein: number;
  carbohydrates: number;
}

// export class FoodItemModel {
//   name: string;
//   types: TypeResource[];
//   typesIds: number[];
//   nutritionalValue: NutritionValueModel;
//   // isLiquid :boolean;
// }
